### PR TITLE
Add test for 'input' block

### LINKED ISSUE #583: input: Check coverage tests

### PR DESCRIPTION
**Pull request by eGavr**
closes #583

### CODE REVIEW

File: common.blocks/input/input.spec.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ describe('input', function() {
     });
 
     describe('val', function() {
+        it('checking of \'getter\' and \'setter\'', function() {
```
Comment (tadatuta) [inline common.blocks/input/input.spec.js]: it's better to use `should [do something]` form. e.g. `should return new value when value changed`

File: common.blocks/input/input.spec.js [hunk 2]
```diff
@@ -15,6 +15,11 @@ describe('input', function() {
...
+            input.setVal('blabla');
```
Comment (tadatuta) [inline common.blocks/input/input.spec.js]: I see it's not used in other tests but we can chain code like this:

``` js
input
    .setVal('blabla')
    .getVal().should.be.equal('blabla');
```

**Comment by gela-d** [comment]
@tadatuta can we merge?

**Comment by tadatuta** [comment]
LGTM

**Pull request merged**